Dockerfile

```diff
@@ -1,9 +1,16 @@
 FROM node:16 AS build
 WORKDIR /app
+
 COPY . .
+
 RUN yarn install --production && \
-  yarn run build
+  yarn build
+
+
+FROM gcr.io/distroless/nodejs:16
```

Comment:
Why would you change the tag? It corresponds to the debian version

Reply:
However, it could be great to move to Debian Bullseye. See https://console.cloud.google.com/gcr/images/distroless/global/nodejs-debian11@sha256:e5e0b00ffae6d134cf45a2ab44d5a8f318d61cb3e46026bfc36056afe99ee659/details?tab=info